paddle/fluid/inference/api/demo_ci/run.sh

```diff
@@ -86,5 +94,25 @@ for WITH_STATIC_LIB in ON OFF; do
       fi
     done
   done
+  
+  # --------tensorrt mobilenet------
+  if [ $USE_TENSORRT == ON -a $TEST_GPU_CPU == ON ]; then
+    rm -rf *
```

Comment:
really dangerous

change to `cd ..; rm -rf build; mkdir build`

Reply:
A new pr will fix this.